arbnode/dataposter/dbstorage/storage.go

```diff
@@ -62,7 +62,7 @@ func (s *Storage) Get(_ context.Context, index uint64) (*storage.QueuedTransacti
 	key := idxToKey(index)
 	value, err := s.db.Get(key)
 	if err != nil {
-		if errors.Is(err, leveldb.ErrNotFound) {
+		if isErrNotFound(err) {
```

Comment:
good call - and are the only errors we care about catching the ones we care about in this function?

```
func isErrNotFound(err error) bool {
	return errors.Is(err, leveldb.ErrNotFound) || errors.Is(err, pebble.ErrNotFound) || errors.Is(err, memorydb.ErrMemorydbNotFound)
}
```

Reply:
I don't know for sure but I believe the implementation of the function that was changed in this PR is supposed to hide the "not found errors" for all implementation, and only this errors.